src/pages/home/report/ReportActionCompose/ReportActionCompose.js

```diff
@@ -179,11 +181,17 @@ function ReportActionCompose({
             return;
         }
         composerRef.current.focus(true);
+
+        // Restore the suggestions if onSelectionChange is not called on focused
+        if (isSelectionChangedOnFocus || !suggestionsRef.current) {
+            return;
+        }
+        suggestionsRef.current.restoreSuggestions();
     };
 
     const isKeyboardVisibleWhenShowingModalRef = useRef(false);
     const restoreKeyboardState = useCallback(() => {
-        if (!isKeyboardVisibleWhenShowingModalRef.current) {
+        if (!isKeyboardVisibleWhenShowingModalRef.current || !shouldFocusInputOnScreenFocus) {
```

Comment:
We don't need to set focus manually if focus is set automatically

Reply:
What problem is this solving?

Reply:
We have composer focused already if `shouldFocusInputOnScreenFocus` is true. In android chrome, onSelectionChange isn't called when we set focus again.
You can reproduce this with the following steps(Android chrome):
1. Open any chat and input `@` to show mentions list
2. Click + button to open menu and tap outside of the menu to close it
3. Verify that mention list doesn't show